src/s11/handlers/s11_MB_resp_handler.c

```diff
@@ -42,6 +43,18 @@ s11_MB_resp_handler(MsgBuffer* message, GtpV2MessageHeader* hdr)
 	mbr_info.ue_idx = hdr->teid;
 	mbr_info.msg_type = modify_bearer_response;
 
+	ModifyBearerResponseMsgData msgData;
```

Comment:
We need a transaction concept at gtp layer. So that only matching transaction response messages are processed and other non matching response messages needs to be dropped.